backend/tests/routes/test_search.py

```diff
@@ -20,7 +20,6 @@
 _TOTAL_DOCUMENT_COUNT = 6
 
 
-@pytest.mark.skip(reason="new test data dump needed")
 @pytest.mark.search
 def test_simple_pagination(test_opensearch, monkeypatch, client):
```

Comment:
Do we still not have enough data to at least get 2 items on a page?

Reply:
we do - i can update the test